src/sentry/static/sentry/app/components/events/sdkUpdates/index.tsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+
+import Alert from 'app/components/alert';
+import {IconUpgrade} from 'app/icons';
+import {tct} from 'app/locale';
+import EventDataSection from 'app/components/events/eventDataSection';
+import {Event} from 'app/types';
+
+import getSuggestion from './getSuggestion';
+
+type Props = {
+  event: Omit<Event, 'sdkUpdates'> & {
+    sdkUpdates: NonNullable<Event['sdkUpdates']>;
+  };
+};
+
+const SDKUpdates = ({event}: Props) => {
+  const {sdkUpdates} = event;
+
+  return (
+    <EventDataSection title={null} type="sdk-updates">
+      {sdkUpdates.map((sdkUpdate, index) => {
+        const suggestion = getSuggestion({suggestion: sdkUpdate, event});
+
+        if (!suggestion) {
+          return null;
```

Comment:
Does this case happen in practice? An empty DataSection will still consume some vertical space. Should the return be above the EventDataSection to emit no DOM?

Reply:
I believe this doesn't happen, but I can not guarantee it....better do as you say...thanks 😊